### PR TITLE
Abp consuming rest api by using static proxy

### PR DESCRIPTION
**Pull request by malikmasis**
closes https://github.com/abpframework/abp/issues/12280

### CODE REVIEW

**Comment by hikalkan** [comment]
Can you ignore the `wwwroot/libs` folder, which is default now. I don't know why it is commiting to github since it seems you are using 5.2.1.
I think it would be better to delete this solution and recreate with the latest abp 5.2.2 by following the [reviesed article](https://github.com/abpframework/abp/commit/2c69850c7ba11492fb1ef7d210771cf501c24879).

File: ABPConsumingRestApiByUsingStaticProxy/README.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+* Add correlation id, current tenant id and the current culture to the request.
+* Properly handles the error messages sent by the server and throws proper exceptions.
+
+##### Differences
+Static generic proxies provide better performance because it doesn't need to run on runtime, but you should **re-generate** once changing API endpoint definition. Dynamic generic proxies don't need **re-generate** again because it works on the runtime but it happens more a bit time. 
+
+Now focus on how to do it,
+Firstly add Volo.Abp.Http.Client NuGet package to your client project:
+````shell
+Install-Package Volo.Abp.Http.Client
```
Comment (enisn) [inline ABPConsumingRestApiByUsingStaticProxy/README.md:150]: Is this required? Doesn't it be included indirectly by BookStore.HttpApi.Client?

**Comment by malikmasis** [comment]
> Can you ignore the `wwwroot/libs` folder, which is default now. I don't know why it is commiting to github since it seems you are using 5.2.1. I think it would be better to delete this solution and recreate with the latest abp 5.2.2 by following the [reviesed article](https://github.com/abpframework/abp/commit/2c69850c7ba11492fb1ef7d210771cf501c24879).

I've tried via version 5.2.2, it seems the same and I've deleted libs and edited .gitignore file.

**Review by hikalkan — APPROVED** [review]

**Review by enisn — COMMENTED** [review]
There is a broken image link, @malikmasis you should consider to fix it or remove it
image

**Pull request merged**